content/en/serverless/step_functions/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+- link: "/serverless/step_functions/installation"
+  tag: "Documentation"
+  text: "Install Serverless Monitoring for AWS Step Functions"
+- link: "/serverless/step_functions/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshoot Serverless Monitoring for AWS Step Functions"
+- link: "/integrations/amazon_step_functions/"
+  tag: "Documentation"
+  text: "AWS Step Functions integration"
+- link: "/serverless/aws_lambda/"
+  tag: "Documentation"
+  text: "Serverless Monitoring for AWS Lambda"
+---
+<div class="alert alert-warning">This feature is in public beta. You can provide feedback by filling out the following <a href="https://docs.google.com/forms/d/e/1FAIpQLSe9BZIxSt4XMKsjZ2xnZCSEGDWbWw75-zMNM6yy3c3QzHx9hg/viewform?usp=sf_link">feedback form</a>.</div>
+
+{{< img src="serverless/step_functions/overview2.jpeg" alt="New Log index" style="width:100%;" >}}
```

Comment:
I don't know why but this images isn't showing up in the preview. In any case I recommend the PNG over JPEG. Also, can you verify the alt text. Looks like it's highlighting an AWS step function span in an flame graph visualization.